packages/components/src/components/Button/Button.test.tsx

```diff
@@ -45,8 +45,8 @@ describe('Button', () => {
       expect(onPressSpy).toBeCalled()
     })
 
-    it('should render label', async () => {
```

Comment:
Is there any harm in having things async even when they don't need to be?

With Link opted to have most async because I was getting confusing errors without it for some tests.

Reply:
I don't think so but we should avoid adding it if there's no async functions within. I think the errors might stem from some of the queries are async functions (e.g. findBy*) and some are not (queryBy*). When we add the jest matchers eslint plugin I think it provides some guidance on which ones should be used.